src/Bridge/Symfony/Validator/EventListener/ValidateListener.php

```diff
@@ -74,7 +74,7 @@ public function onKernelView(GetResponseForControllerResultEvent $event)
         }
 
         $violations = $this->validator->validate($data, null, $validationGroups);
-        if (0 !== count($violations)) {
+        if ($violations) {
```

Comment:
This evaluates to `true` even if there are no violations. `count` isn't that useless here.